src/libraries/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -595,7 +595,17 @@ public sealed partial class Graphics : System.MarshalByRefObject, System.Drawing
         public static System.Drawing.Graphics FromHwndInternal(System.IntPtr hwnd) { throw null; }
         public static System.Drawing.Graphics FromImage(System.Drawing.Image image) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
+#if NET50_OBSOLETIONS
+        [System.ObsoleteAttribute(Obsoletions.GetContextInfoMessage, DiagnosticId = Obsoletions.GetContextInfoId, UrlFormat = Obsoletions.SharedUrlFormat)]
+#else
+        [System.ObsoleteAttribute(Obsoletions.GetContextInfoMessage)]
+#endif
```

Comment:
The pattern we have used, like with other attribute values, is to expand/hard-code the strings in the ref files and use the `Obsoletions` fields in src files.

```suggestion
#if NET50_OBSOLETIONS
        [System.ObsoleteAttribute("Use the Graphics.GetContextInfo overloads that accept arguments for better performance and fewer allocations.", DiagnosticId = "SYSLIB0016", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
#endif
```

We've also not been obsoleting down-level since users cannot suppress specific obsoletions down-level.